src/engine/cache/file-cache.js

```diff
@@ -122,24 +130,22 @@ class FileCache extends BaseCache {
       return null
     }
 
-    const sortedFiles = fileNames
-      .map(async (fileName) => {
-        // Retrieve file state to retrieve the oldest one from the cache
-        let fileState = null
+    const fileStats = []
+    await fileNames.reduce((promise, fileName) => promise.then(
+      async () => {
         try {
-          // Error triggered when a file is being written (from a South)
-          fileState = await fs.stat(path.resolve(this.fileFolder, fileName))
+          const stat = await fs.stat(path.resolve(this.fileFolder, fileName))
+          fileStats.push({
+            path: path.resolve(this.fileFolder, fileName),
+            timestamp: stat.mtime.getTime(),
+          })
         } catch (error) {
           this.logger.error(error)
```

Comment:
a comment explaining why this can happen and why silently ignoring it is the right way to go would be more than helpful